docs/intro.md

```diff
@@ -227,6 +231,107 @@ Please refer to the [project
 documentation](https://ogmios.dev/getting-started/building/#-documentation) for details on how to
 run and configure ogmios.
 
+### Configuration
```

Comment:
I am aware that there are some additional config files of relevance when dealing with wizards, but I wanted to keep this those out of scope here. They can (and maybe should?) be documented elsewhere in sections about the respective wizards